www/src/components/shell/TerminalSidebar.js

```diff
@@ -465,28 +471,39 @@ function Cheatsheet() {
             title="Commands"
             onClick={() => setActiveSection(SECTION_COMMANDS)}
           />
-          <CheatsheetItem
-            title="Debugging"
-            onClick={() => setActiveSection(SECTION_DEBUGGING)}
-          />
-          <CheatsheetItem
-            title="Workspace"
-            onClick={() => setActiveSection(SECTION_WORKSPACE)}
-          />
+          {/* TODO: enable when other sections design will be finished */}
+          {/* <CheatsheetItem */}
+          {/*  title="Debugging" */}
+          {/*  onClick={() => setActiveSection(SECTION_DEBUGGING)} */}
+          {/* /> */}
+          {/* <CheatsheetItem */}
+          {/*  title="Workspace" */}
+          {/*  onClick={() => setActiveSection(SECTION_WORKSPACE)} */}
+          {/* /> */}
         </Div>
         <Div
           width={sidebarWidth}
           flexShrink={0}
           paddingVertical="medium"
         >
-          <Button
-            small
-            tertiary
-            onClick={() => setActiveSection(null)}
-            marginLeft="medium"
+          <Flex
+            direction="row"
+            align="center"
+            marginLeft="large"
+            fontSize="small"
           >
-            Back to cheatsheet
-          </Button>
+            <Icon
+              icon={(
+                <ArrowLeftIcon size={16} />
+              )}
+              onClick={() => setActiveSection(null)}
+            />
+            <P
+              marginLeft="xsmall"
+            >
+              Back to cheatsheet
+            </P>
+          </Flex>
```

Comment:
I think the text should be included in this button, and button should align with text below.
![cheatsheet-markup](https://user-images.githubusercontent.com/85062/178048203-6d02790f-5d26-4efe-99f2-7e193c03b91f.png)
.

Reply:
Oh yeah. Makes sense :slightly_smiling_face: